wiki/Installation/en.md

```diff
@@ -2,7 +2,7 @@
 
 <!-- *See also: [Installation/OS X](/wiki/Installation/OS_X) and [Installation/Linux](/wiki/Installation/Linux)* -->
```

Comment:
```suggestion
*See also: [Installation/Mac_OS](/wiki/Installation/Mac_OS)* <!-- and [Installation/Linux](/wiki/Installation/Linux)* -->
```

Reply:
I think it is actually spelt as `macOS` with lowercase M and no space.

Reply:
hasn't been addressed